src/components/MapsSection/MapsSection.tsx

```diff
@@ -104,40 +68,25 @@ const MapsSection = ({
           <ButtonsWrapper>
             <ExpandBox
               href={`/map?name=${currentVisu.id}`}
```

Comment:
Fix the redirection by name
```suggestion
              href={`/map?id=${currentVisu.id}`}
```